src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card className="text-left">
+      <CardContent>
+        <div class="row">
+          <h3>{title}</h3>
+        </div>
+        <div className="d-flex col-12">
+          <div class="row">
+            <div id="dg_main-content">
```

Comment:
we will want to remove this id, since it will be repeated.

Reply:
I think we probably want something like

```jsx
<div className="col-3">
  <img ...
</div>
<div className="col-9">
  {/* card summary contents or however you want to describe it*/}
</div>
```

This is jsut an example on desktop, well want to make sure this responds, I also think this may help us with teh image sizing issue (partially).

Reply:
so the published date and author need to be inside of a row and col to line up as well so minor change to this but general idea stands.